src/test_utils/rpc/mod.rs

```diff
@@ -86,3 +87,50 @@ pub async fn start_kakarot_rpc_server(katana: &Katana) -> Result<(SocketAddr, Se
     )
     .await?)
 }
+
+/// Represents a builder for creating JSON-RPC requests.
+/// Taken from https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc-builder/tests/it/http.rs
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct RawRpcParamsBuilder {
+    method: Option<String>,
+    params: Vec<Value>,
+    id: i32,
+}
+
+impl RawRpcParamsBuilder {
+    /// Sets the method name for the JSON-RPC request.
+    pub fn method(mut self, method: impl Into<String>) -> Self {
```

Comment:
what do you think of having this as `new`? You can't really have an rpc without a method right?

Reply:
Not sure because the basic flow is like:

```rust
RawRpcParamsBuilder::default()
                .method("alchemy_getTokenBalances")
                .add_param(eoa_address)
                .add_param([erc20_address])
                .build()
```

In order to:
1. Initialize the empty builder with just the ID and an empty array of parameters.
2. Add a method
3. Add parameters
4. Build all of that

So if we rename this as `new` it could be a bit strange to have something like:
```rust
RawRpcParamsBuilder::default()
                .new("alchemy_getTokenBalances")
                .add_param(eoa_address)
                .add_param([erc20_address])
                .build()
```

no? We would integrate the logic of `default` inside the new but I have the impression that the ID set to 1 should be in the default method, right?

Reply:
I was more seeing it like 
```
RawRpcParamsBuilder::new("alchemy_getTokenBalances")
                .add_param(eoa_address)
                .add_param([erc20_address])
                .build()
```

Reply:
Just updated, let me know :)